.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "vendor/http-parser"]
```

Comment:
Looks like we no longer need the `http-parser` submodule, right?

Reply:
Hi! Actually it is used to parse urls, as llhttp don't have this feature!

Reply:
`uparser.pxd` is used only to extern url parsing methods from `http_parser.h`, and `url_parser..pyx` contains previous code related to url parsing. I couldn't extern both `llhttp.h` and `http_parser.h` on the same file as they have naming conflicts